### ISSUE TITLE
Update tests

### ISSUE THREAD

**Issue by mojoaxel** [visjs/vis-util#6]
It would be nice and more secure if we move code in the future if every function would have its own test - ideal in its own test-file.

**Comment by Thomaash** [comment]
:fireworks: Our test coverage is all green. :fireworks:

```diff
+ Statements   : 82.51% ( 453/549 )
+ Branches     : 81.21% ( 376/463 )
+ Functions    : 81.93% ( 68/83 )
+ Lines        : 82.59% ( 446/540 )
```

PS: There are still two unmerged tests to make it even better.

**Comment by mojoaxel** [comment]
It would have been better to have these test coverage before the TS rewrite! 
But better late than never :rofl:

**Comment by Thomaash** [comment]
Yeah, that's what I plan to do in vis-network. Tests first, TS later.
